### ISSUE TITLE
Document orphaned resources, or allow removal?

### ISSUE THREAD

**Issue by SeanBannister** [aws/awsmobile-cli#104]
When a user runs `awsmobile delete` a number of resources aren't deleted and are left orphaned in the account. Could I suggest that these are documented somewhere so the user knows what to delete manually. It appears to be [dynamodb](https://console.aws.amazon.com/dynamodb/home) and [s3](https://s3.console.aws.amazon.com/s3/home) are there any other resources users should be aware of?

An alternative could be providing an extra parameter on the `awsmobile delete` command that also deletes these resources. This is handy in development when spinning up multiple environments.

**Comment by UnleashedMind** [comment]
Thanks for the feedbacks @SeanBannister. 
We are aware of that issue, and we have notified our mobile hub team about it, will notify you when there is any update. 
You suggestion about the extra parameter in the cli will also be considered.

**Comment by gentleShark** [comment]
agreed. it looks like appsync is the only resources that gets removed and everything else is left orphaned.

**Comment by stevenpovlitz** [comment]
Related to this discussion: CloudFormation can be used to build / tear down resources, as long as MobileHub is not used. 

https://twitter.com/dabit3/status/1022192837683634176
